loki/transformations/single_column/scc_cuf.py

```diff
@@ -292,10 +292,10 @@ def kernel_cuf(self, routine, horizontal, vertical, block_dim,
         loop_map = {}
         for loop in FindNodes(ir.Loop).visit(routine.body):
             if loop.variable == self.block_dim.index or loop.variable.name.lower()\
-                    in [_.lower() for _ in self.block_dim._aliases]:
+                    in [_.lower() for _ in self.block_dim.sizes]:
```

Comment:
Looking at this pattern, there might be an argument to enforce lower case for all these epxressions in `Dimension` itself.

Reply:
No, I think this should really be mopped up by symbol-string-comparison. I'll adjust in the transformation as this makes it cleaner to read.